### DISCUSSION TITLE
nginxproxymanager : unable to create SSL cert for godaddy

### DISCUSSION THREAD

**Discussion by ghost** [tteck/Proxmox#2346] [Help Needed] [answered]
Hello,

I am first time user of NginxproxyManager. I have a domain through "GoDaddy". I am trying to generate SSL certificate using Let's Encrypt - > DNS Challenge option. I keep getting the following error

certbot: error: unrecognized arguments: --dns-godaddy-credentials /etc/letsencrypt/credentials/credentials-13 --dns-godaddy-propagation-seconds 120

I have installed NginxproxyManager on a Proxmox LXC Container using this script -> "https://github.com/tteck/Proxmox/raw/main/ct/nginxproxymanager.sh".

Just for testing purposes, I tried to generate SSL certificate for one of my duckdns.org domain and that worked fine... but for some reason godaddy isn't working.

Any help would be really appreciated.

Thanks.

**Comment by tteck** [comment, marked as answer]
> 🚨 Since there are hundreds of Certbot instances, it's necessary to install the specific Certbot of your preference.

https://pypi.org/search/?q=certbot+godaddy&o=

**Reply by ghost** [reply]
Hey... thanks very much...

I installed using the command "pip install certbot-dns-godaddy". I didn't create any vene or anything. And it seems to be working now. Is this the right way or was I suppose to uninstall previous version of certbot or something?

Sorry I am very new to NginxProxymanger, Certbot etc... so please excuse me for my dumb questions... :)

Thanks.
